nengo/spa/actions.py

```diff
@@ -154,7 +154,11 @@ class Actions(object):
     def __init__(self, *args, **kwargs):
         self.actions = None
         self.args = args
-        self.kwargs = kwargs
+        self.kwargs = OrderedDict(kwargs)
```

Comment:
Yeah that sounds good. The main idea is it should be all-or-nothing. So either clearly unordered or clearly ordered, otherwise people come to rely on something that seems defined, only to have things confusingly fall apart later when things get more complex and difficult to debug. :+1: